Sources/Stats/mc_stats.cc

```diff
@@ -173,10 +173,15 @@ Eigen::VectorXcd product_sv(Eigen::Ref<const Eigen::VectorXcd> s_values,
   return product;
 }
 
-void SubtractMean(Eigen::Ref<RowMatrix<Complex>> v_values) {
+Eigen::VectorXcd ComputeMean(Eigen::Ref<RowMatrix<Complex>> v_values) {
```

Comment:
we don't need these functions anymore, right?

Reply:
they are still called from python in `stats.subtract_mean`. Can easily be deleted though

Reply:
I got rid of those.